.github/workflows/deploy.yml

```diff
@@ -119,7 +119,7 @@ jobs:
           export TF_VAR_paas_app_docker_image=${{ env.DOCKERHUB_REPOSITORY_BASE }}${{ env.DOCKERHUB_ENV }}:${{ env.TAG }}
           cd terraform/app
           terraform init -reconfigure -input=false -backend-config="bucket=paas-s3-broker-prod-lon-7f2ca242-9929-4662-a79c-c454ea56ea7b"
-          terraform apply -input=false -auto-approve -var-file ../workspace-variables/dev.tfvars -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_DEV}}"}'
+          terraform apply -input=false -auto-approve -var-file ../workspace-variables/dev.tfvars -var='secret_paas_app_env_values={"RAILS_MASTER_KEY":"${{secrets.RAILS_MASTER_KEY_DEV}}"}' -var='environment=${{github.event.inputs.environment}}' -var='secret_paas_app_env_values={"SYSLOG_DRAIN_URL":"${{secrets.SYSLOG_DRAIN_URL}}"}'
```

Comment:
We should pass in a `var syslog_drain_url='=${{secrets.SYSLOG_DRAIN_URL}}'`, although maybe it should be called `logstash_url`

Reply:
We don't want to sent the environment here, that was omitted deliberately